maven-spy/pom.xml

```diff
@@ -67,8 +67,8 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <source>1.6</source>
-          <target>1.6</target>
+          <source>1.8</source>
+          <target>1.8</target>
```

Comment:
no that's wrong!
We must use release! 
see PR https://github.com/jenkinsci/pipeline-maven-plugin/pull/703/
I asked in this PR what version level we want.

Reply:
I know !!!! First merge #703, then rebase and fix here to switch from source/target to release